packages/devtools_app/lib/src/ui/fake_file/file_io.dart

```diff
@@ -3,11 +3,17 @@
 // found in the LICENSE file.
 
 abstract class FileIO {
+  /// Create file in a directory (default Desktop in \tmp).
+  // TODO(terry): Better directory for Flutter Desktop when API available.
+  // TODO(terry): Flutter Web/HTML port code to create file in Download directory.
   void writeStringToFile(String filename, String contents);
 
+  /// Returns content of filename or null if file is unknown or content empty.
   String readStringFromFile(String filename);
 
+  /// List of files (basename only).
   List<String> list();
 
+  /// Delete exported files created for testing only.
```

Comment:
if this is only for testing, can you add a `@visibleForTesting` annotation. Or even better, perhaps you could create an extension method to do this within the test file?